ext/iconv/tests/iconv_strpos_error_001.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+iconv_strpos() - test against PHP's defaults internal encoding. Non windows systems.
+--EXTENSIONS--
+iconv
+--SKIPIF--
+<?php
+if (substr(PHP_OS, 0, 3) == "WIN" ) {
+    die("skip test is not for windows systems only");
+}
+?>
```

Comment:
I think the `--SKIPIF--` section can be removed:
```suggestion
```
Also the test title should be adjusted accordingly.

Reply:
Totally forgot about that